### DISCUSSION TITLE
Replace VolumeDB to Volume 0 to 100

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9848] [Editor]
[Deleted]

**Comment by dalexeev** [comment]
[Audio buses - Decibel scale](https://docs.godotengine.org/en/stable/tutorials/audio/audio_buses.html#decibel-scale):

> Godot's sound interface is designed to meet the expectations of sound design professionals. To this end, it primarily uses the decibel scale.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/1884.
